src/chrome/content/rules/Boeing-Employees-Credit-Union.xml

```diff
@@ -23,7 +23,7 @@
 
 	<!--	Some pages redirect to http.  There may be more that don't.	-->
 	<rule from="^http://www\.becu\.org/($|contact-us\.aspx|css/|Default\.aspx|flash/|images/|js/|mobile-online-banking/|pdfsource/|who-is/becu-is-you\.aspx)"
-		to="https://www.becu.org/$1s/" />
+		to="https://www.becu.org/$1/" />
```

Comment:
Is the trailing / still wanted?

https://www.becu.org/who-is/becu-is-you.aspx works

https://www.becu.org/who-is/becu-is-you.aspx/ gives a 404